.github/workflows/ci.yml

```diff
@@ -333,6 +278,61 @@ jobs:
           path: ${{ github.workspace }}/tests/${{ steps.smoke-tests.outputs.test-results-name }}.html
         if: always()
 
+  build-docker:
+    name: Build Docker OSS
+    needs: smoke-tests
+    strategy:
+        fail-fast: false
+        matrix:
+          image: [debian, alpine]
+          platforms: ["linux/arm, linux/arm64, linux/amd64, linux/ppc64le, linux/s390x"]
+          include:
+            - image: ubi
+              platforms: "linux/arm64, linux/amd64, linux/ppc64le, linux/s390x"
+    uses: ./.github/workflows/build-oss.yml
+    with:
+      platforms: ${{ matrix.platforms }}
+      image: ${{ matrix.image }}
+    secrets: inherit
+
+  build-docker-plus:
+    name: Build Docker Plus
+    needs: build-docker
+    strategy:
+        fail-fast: false
+        matrix:
+          image: [debian-plus, alpine-plus]
+          platforms: ["linux/arm64, linux/amd64"]
+          target: [goreleaser, aws]
+          include:
+            - image: ubi-plus
+              platforms: "linux/arm64, linux/amd64, linux/s390x"
+              target: goreleaser
+    uses: ./.github/workflows/build-plus.yml
+    with:
+      platforms: ${{ matrix.platforms }}
+      image: ${{ matrix.image }}
+      target: ${{ matrix.target }}
+    secrets: inherit
+
+  build-docker-nap:
+    name: Build Docker NAP
+    needs: build-docker-plus
```

Comment:
Why do we need to build the 3 types of images consecutively instead of concurrently?

Reply:
It was to limit the number of cache access and get fewer `429`s from GitHub. But it seems like it's not doing much anyway, we can try to remove it again...